### DISCUSSION TITLE
XPF for series of apps?

### DISCUSSION THREAD

**Discussion by mysteryx93** [AvaloniaUI/Avalonia#14204] [General]
Let's say a company has a whole series of WPF apps, and instead of porting them, they want to use XPF. The fee is pretty heavy, 5000 euro per app per platform. If there are many apps, is there a bundle price or the fee would be multiplied for each app?

And is there a point in paying license for Windows, since the WPF code already runs on Windows?

**Comment by MikeCodesDotNET** [comment]
Thanks for your interest in Avalonia XPF. I want to clarify a couple of points regarding our licensing model:

* **Windows License Included:** We understand the redundancy in purchasing a separate Windows license when the WPF code already runs on Windows. Therefore, a Windows license is free of charge when buying a license for other platforms such as macOS or Linux.
* **Flexible Licensing Options:** Recognising the diverse needs of our customers, especially those with numerous WPF applications, we offer a range of licensing options. This includes site licenses, ideal for large enterprises with lots of apps. Additionally, we provide discounts for purchases involving multiple apps or platforms, ensuring a cost-effective solution tailored to your specific requirements.

We are committed to providing flexible and fair licensing terms that cater to all business needs. Please don't hesitate to reach out for further details or to discuss our licensing.

**Reply by mysteryx93** [reply]
Thanks! I'm trying to bring you a pretty big client so let's see what they say.

By the way, does XPF allow for development of WPF applications on Linux?

And thanks for the amazing work you're doing

**Reply by MikeCodesDotNET** [reply]
@mysteryx93 great to hear! 

We've found the best way to convince customers is to get them to experience their apps on macOS and Linux. Perhaps have them [sign up for a trial](https://avaloniaui.net/xpf/interest) to try themselves?

**Reply by mysteryx93** [reply]
What about WPF development on Linux?

**Reply by MikeCodesDotNET** [reply]
> What about WPF development on Linux?

@mysteryx93 it works well.
